### PR TITLE
aaron/Contract confirmation + information

### PR DESCRIPTION
**Pull request by aaimio**

### CODE REVIEW

File: src/javascript/app/pages/trade/digit_ticker.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ const DigitTicker = (() => {
         el_container.classList.remove('invisible');
         adjustBoxSizes();
         current_spot = quote.substr(-1);
-        if (el_mask) el_mask.innerText = `${current_tick_count} / ${total_tick_count}`;
+        if (el_mask) el_mask.innerText = `${current_tick_count}/${total_tick_count}`;
```
Comment (negarn) [inline src/javascript/app/pages/trade/digit_ticker.js]: could you undo these spacing changes please? let's keep it same as master if possible

File: src/sass/_common/base/constants.scss [hunk 2]
```diff
@@ -38,18 +37,8 @@ $COLOR_LIGHT_GRAY    : $fillColor;     // #f2f2f2
 $COLOR_GRAY          : $borderColor;   // #dedede
 $COLOR_DARK_GRAY     : #44484c;
 $COLOR_GREEN         : $positiveColor; // #2e8836
-
-$COLOR_NETWORK_STATUS_ONLINE: #4bb4b3;
-$COLOR_NETWORK_STATUS_OFFLINE: #ec3f3f;
-$COLOR_NETWORK_STATUS_BLINKER: #4bb4b3;
-$COLOR_BORDER_GRAY: #f2f3f4;
-$COLOR_BALLOON_GRAY: #d6dadb;
-$COLOR_HOVER_FULLSCREEN_GRAY: #e6e9e9;
```
Comment (negarn) [inline src/sass/_common/base/constants.scss]: why delete these? I'm using some of them, and the codebase needs the others too

File: src/sass/app/components/peek_box.scss [hunk 3]
```diff
@@ -203,10 +202,10 @@ $COLOR_PEEK_LIGHT_GRAY: #c2c2c2;
...
         justify-content: center;
         font-weight: bold;
-        border-right: 1px solid $COLOR_GRAY;
-        border-top: 1px solid $COLOR_GRAY;
-        border-bottom: 1px solid $COLOR_GRAY;
-        color: $COLOR_PEEK_LIGHT_GRAY;
+        border-right: 1px solid $color-grey-5;
+        border-top: 1px solid $color-grey-5;
+        border-bottom: 1px solid $color-grey-5;
+        color: $color-grey-5;
```
Comment (negarn) [inline src/sass/app/components/peek_box.scss]: can you use them this way? or you need to use the var version from themes?

File: src/sass/app/sell_popup.scss [hunk 4]
```diff
@@ -115,7 +117,7 @@
                     }
                 }
                 td, td[data-balloon] {
-                    border-bottom: 1px solid $COLOR_LIGHT_GRAY;
+                    border-bottom: 1px solid $color-grey-5;
```
Comment (negarn) [inline src/sass/app/sell_popup.scss]: same here

File: src/sass/app/sell_popup.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/* styline-disable */
```
Comment (negarn) [inline src/sass/app/sell_popup.scss]: what is styline? why do we need to disable it?

File: src/sass/app/trade.scss [hunk 6]
```diff
@@ -49,6 +49,7 @@
 }
 
 #trading_socket_container {
+    color: $color-black-1;
```
Comment (negarn) [inline src/sass/app/trade.scss]: same here

File: src/sass/_common/base/constants.scss (hunk 2 above)
Comment (aaimio) [reply src/sass/_common/base/constants.scss]: - I've removed `$COLOR_NETWORK_STATUS_BLINKER` and `$COLOR_HOVER_FULLSCREEN_GRAY` as they're not from Binary Static originally, I've updated their usages by using CSS vars from `theme.scss`. 
- `$COLOR_BORDER_GRAY` and `$COLOR_BALLOON_GRAY` should still be available, however we should try to use CSS vars where possible.

File: src/javascript/app/pages/trade/digit_ticker.js (hunk 1 above)
Comment (aaimio) [reply src/javascript/app/pages/trade/digit_ticker.js]: I've removed it as per the design, but we could revert. 👍

File: src/sass/app/components/peek_box.scss (hunk 3 above)
Comment (aaimio) [reply src/sass/app/components/peek_box.scss]: You're right, I should use CSS vars here. 👍

File: src/sass/app/sell_popup.scss (hunk 4 above)
Comment (aaimio) [reply src/sass/app/sell_popup.scss]: 👍

File: src/sass/app/sell_popup.scss (hunk 5 above)
Comment (aaimio) [reply src/sass/app/sell_popup.scss]: We don't, it's remnant that needs to be removed. 👍

File: src/sass/app/trade.scss (hunk 6 above)
Comment (aaimio) [reply src/sass/app/trade.scss]: 👍

File: src/sass/_common/base/constants.scss (hunk 2 above)
Comment (negarn) [reply src/sass/_common/base/constants.scss]: I think we shouldn't delete them, even if we don't use them let's keep them. we want to change the codebase as less as possible

File: src/javascript/app/pages/trade/digit_ticker.js (hunk 1 above)
Comment (negarn) [reply src/javascript/app/pages/trade/digit_ticker.js]: oh, right, it's in string :) let's keep them, my bad

File: src/sass/_common/base/constants.scss (hunk 2 above)
Comment (aaimio) [reply src/sass/_common/base/constants.scss]: They were added as part of our changes (Brandon's task).

Comment (negarn) [reply src/sass/_common/base/constants.scss]: ah, I see! I had no idea :) it's alright then, thank you

File: src/sass/_common/base/constants.scss [hunk 7]
```diff
@@ -38,6 +37,8 @@ $COLOR_LIGHT_GRAY    : $fillColor;     // #f2f2f2
 $COLOR_GRAY          : $borderColor;   // #dedede
 $COLOR_DARK_GRAY     : #44484c;
 $COLOR_GREEN         : $positiveColor; // #2e8836
+$COLOR_BORDER_GRAY   : #f2f3f4;
+$COLOR_BALLOON_GRAY  : #d6dadb;
```
Comment (negarn) [inline src/sass/_common/base/constants.scss]: I think these two colours aren't being used anywhere

Comment (negarn) [reply src/sass/_common/base/constants.scss]: I've removed any missed usage in my chart control PR https://github.com/deriv-com/binary-static/pull/19

Comment (aaimio) [reply src/sass/_common/base/constants.scss]: I've removed these.

**Pull request merged**